docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #160.                PL[i] = PL[i-1]
</pre><p>If <code>i</code> is pruned, we may just set the value of <code>PL[i] </code> to <code>np.inf </code> so to make it more intuitive and readable. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Since `PL==np.inf` has a special meaning (i.e. the subsequence has at least one non-finite value or the subsequence has no left neighbor), it is better to keep it as is.